### PR TITLE
Removed capabilities filtering and public capability constants

### PR DESCRIPTION
**Pull request by oldalton**

### CODE REVIEW

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m [hunk 1]
```diff
@@ -590,7 +589,7 @@ - (void)acquireTokenInteractive:(id)sender
 
     if (_enableClientCapabilities.selectedSegmentIndex == 1)
     {
-        capabilities = @[AD_CLIENT_CAPABILITY_LLT];
+        capabilities = @[@"cp1"];
```
Comment (jasoncoolmax) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m]: Just curious, llt is not no longer allowed for the manual testing? :)

File: ADAL/src/utils/ADClientCapabilitiesUtil.h [hunk 2]
```diff
@@ -25,8 +25,6 @@
 
 @interface ADClientCapabilitiesUtil : NSObject
 
-+ (NSArray<NSString *> *)knownCapabilities:(NSArray<NSString *> *)allCapabilities;
-
 /*
```
Comment (jasoncoolmax) [inline ADAL/src/utils/ADClientCapabilitiesUtil.h]: I like the change as we don't need to update library if server come up with a new capability :)

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m (hunk 1 above)
Comment (oldalton) [reply Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m]: "llt" got changed to "cp1" :)

File: ADAL/src/utils/ADClientCapabilitiesUtil.h (hunk 2 above)
Comment (oldalton) [reply ADAL/src/utils/ADClientCapabilitiesUtil.h]: 👍

**Pull request merged**